forum_leaderboard/static/html/forum_leaderboard.html

```diff
@@ -0,0 +1,8 @@
+<div class="forum_leaderboard_block">
+<h2>{{ display_name }}</h2>
+<ul>
+    {% for thread in threads %}
+        <li>{{ thread.title }} (Score: {{ thread.votes.point }})</li>
```

Comment:
I guess this will be done later but it would be nice to show the thread author's username and even better to link to the thread.